modules/compliance-supported-profiles.adoc

```diff
@@ -86,13 +86,13 @@ The Compliance Operator provides the following compliance profiles:
 |0.1.47+
 |link:https://www.pcisecuritystandards.org/document_library?document=pci_dss[PCI Security Standards &#174; Council Document Library]
 |`x86_64`
-
+ `ppc64le`
```

Comment:
I know this still renders correctly, but include an empty line below `ppc64le`  so that table entries do not run together.

Reply:
@prb112 just want to confirm with you this is okay to add back in now.

Reply:
it's OK to add back ppc64le. Thank you